ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -1,12 +1,112 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
+import { useGasFeeEstimates } from '../../../hooks/useGasFeeEstimates';
+import { I18nContext } from '../../../contexts/i18n';
+
 import Typography from '../../ui/typography/typography';
 import { TYPOGRAPHY } from '../../../helpers/constants/design-system';
 import InfoTooltip from '../../ui/info-tooltip';
 
-export default function GasTiming({ text, tooltipText, attitude }) {
+function toHumanReadableTime(milliseconds) {
+  const rtf = new Intl.RelativeTimeFormat(undefined, {
+    localeMatcher: 'best fit',
+    numeric: 'always',
+    style: 'long',
+  });
+
+  return rtf.format(milliseconds / 1000, 'second');
```

Comment:
I'm not convinced this API will work -- it prefixes `in` on its time estimates, which is problematic.

Reply:
```
rtf.formatToParts(100, "day");
// > [{ type: "literal", value: "in " },
// >  { type: "integer", value: "100", unit: "day" },
// >  { type: "literal", value: " days" }]
```
you could do something like this and use the pieces, just insert the `<` symbol but that won't be internationalized (might be in the wrong place?)